Resources/Prototypes/Catalog/uplink_catalog.yml

```diff
@@ -1325,19 +1325,16 @@
       tags:
       - NukeOpsUplink
 
-#- type: listing # DeltaV - Remove syndicats
-#  id: UplinkMobCatMicrobomb
-#  name: uplink-mobcat-microbomb-name
-#  description: uplink-mobcat-microbomb-desc
-#  icon: { sprite: Objects/Devices/communication.rsi, state: old-radio-syndicat }
-#  productEntity: ReinforcementRadioSyndicateSyndiCat
-#  discountCategory: usualDiscounts
-#  discountDownTo:
-#    Telecrystal: 3
-#  cost:
-#    Telecrystal: 6
-#  categories:
-#    - UplinkAllies
+- type: listing # DeltaV - Readd Syndicat with C4
+  id: UplinkMobCatMicrobomb
+  name: uplink-mobcat-microbomb-name
+  description: uplink-mobcat-microbomb-desc
+  icon: { sprite: Objects/Devices/communication.rsi, state: old-radio-syndicat }
+  productEntity: ReinforcementRadioSyndicateSyndiCat
```

Comment:
Update `productEntity` to `ReinforcementRadioSyndicateSyndiCatDV` from [this comment](https://github.com/DeltaV-Station/Delta-v/pull/4731/files#r2574311736)